### PR TITLE
News Updates for Summer 2023.

### PR DESCRIPTION
**Pull request by whitwham**
Release news.

### CODE REVIEW

**Comment by whitwham** [comment]
This is for review.  I will keep adding to it until the release.

**Comment by jkbonfield** [comment]
I'll have a read through, but the formatting differs to normal slighty.   Eg compared

```
* Speed up calmd's slow handling of non-position-sorted data by adding caching.
  This uses more memory but is only activated when needed.
```

vs

```
* Speed up calmd's slow handling of non-position-sorted data by adding caching.
This uses more memory but is only activated when needed.
```

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,94 @@
...
+* Fix bug in split output format.  Now SAM and CRAM format can chosen as well
+  as BAM.  Also a documentation change, see below.
+  (PR#1821)
+
+* Add error checking to view -e filter expression code.  Invalid expressions
+  were not returning an error code.
+  (PR#1833, fixes #1829.  Reported by Steve Huang)
+
+* Fix reheader CRAM output version.  Sets the correct CRAM output version for
+  non-3.0 CRAMS.
```
Comment (jmarshall) [inline NEWS:66]:
```suggestion
  non-3.0 CRAMs.
```

File: NEWS [hunk 2]
```diff
@@ -1,6 +1,94 @@
...
+* Expand the default filtering information on the mpileup manpage.
+  (PR#1802, fixes #1801.  Reported by gevro)
+
+* Add an explanation of the default behaviour of split files on generating
+  a file for reads with missing or unrecognised RG tags.  Also a small bug fix,
+  see above.
+  (PR#1821, fixes #1817.  Reported by Steve Huang)
+
+* In the INSTALL instructions, switched back to openssl for Alpine.  This
+  matches the current Alpine Linux practise.
```
Comment (jmarshall) [inline NEWS:80]:
```suggestion
  matches the current Alpine Linux practice.
```

File: NEWS [hunk 3]
```diff
@@ -1,6 +1,94 @@
...
+  (PR#1837, see htslib#1591.  Reported by John Marshall)
+
+* Fix various typos caught by lintian parsers.
+  (PR#1877.  Thanks to �tienne Mollier)
```
Comment (jmarshall) [inline NEWS:84]: Accented letter should be UTF-8 encoded.

File: NEWS [hunk 4]
```diff
@@ -1,6 +1,94 @@
...
+  were not returning an error code.
+  (PR#1833, fixes #1829.  Reported by Steve Huang)
+
+* Fix reheader CRAM output version.  Sets the correct CRAM output version for
+  non-3.0 CRAMS.
+  (PR#1868, fixes #1866.  Reported by John Marshall)
+
+Documentation:
+
+* Expand the default filtering information on the mpileup manpage.
```
Comment (jmarshall) [inline NEWS:71]:
```suggestion
* Expand the default filtering information on the mpileup man page.
```

**Pull request merged**